### PR TITLE
Translations

### PR DESCRIPTION
**Pull request by dbillionlabs**
Some translations to Spanish

### CODE REVIEW

**Comment by GrahamCampbell** [comment]
Ping @joecohens.

**Comment by dbillionlabs** [comment]
Another file left to translate, you'll have it tomorrow.

**Comment by GrahamCampbell** [comment]
Thanks. We'll wait for that and then I'll squash and merge this once Joe is happy. ;)

File: resources/lang/es/dashboard.php [hunk 1]
```diff
@@ -72,9 +72,9 @@
                 'failure' => 'Algo salió mal con el grupo del componente.',
             ],
             'edit' => [
-                'title'   => 'Edit a component group',
-                'success' => 'Component group updated.',
-                'failure' => 'Something went wrong with the component group.',
+                'title'   => 'Editar un grupo de componentes',
+                'success' => 'Grupo de componentes actualizado.',
+                'failure' => 'Algo ha ido mal con el grupo de componentes.',
```
Comment (joecohens) [inline resources/lang/es/dashboard.php]: @dbillionlabs Podemos cambiar el 'Algo ha ido mal' por 'Algo salió mal' igual que los mensajes que tenemos. Gracias!

Comment (dbillionlabs) [inline resources/lang/es/dashboard.php]: sure

Comment (joecohens) [inline resources/lang/es/dashboard.php]: Thanks ;)

**Comment by dbillionlabs** [comment]
Here it's.

**Comment by GrahamCampbell** [comment]
Merged. Thank you. :)

**Comment by joecohens** [comment]
:+1:

**Pull request closed without merging**